flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -79,6 +79,10 @@ public class JobDetailsInfo implements ResponseBody {
 
     public static final String FIELD_NAME_JSON_PLAN = "plan";
 
+    public static final String FIELD_NAME_STREAM_GRAPH_JSON_PLAN = "stream-graph-plan";
```

Comment:
Could you add some comments to describe its structure?

Reply:
Maybe name it directly as `stream-graph` to avoid the redundant `plan`.  StreamGraph already indicates it is a plan.

Reply:
Thanks @zhuzhurk for the comments! It is true that `stream-graph-plan` has some redundancy in its naming, I renamed this field to `stream-graph` and added notes explaining the contents of the field.